index.html

```diff
@@ -21,7 +27,7 @@ <h1 class="title">Michael<br/>Olusola</h1>
   </section>
   <section class="projects">
     <h2 class="project-heading">Projects</h2>
-    <article class="project-card">
+    <article class="project-card-1">
```

Comment:
- [ ] you have an overflow in your website because of some big fixed-width images, kindly try to fix it so your website looks better

![demoover](https://user-images.githubusercontent.com/89282221/187048000-5af26d16-3d49-45ab-bd12-9321f5942cc5.gif)